src/core/precomp.h

```diff
@@ -17,6 +17,7 @@
 #pragma warning(disable:28931) // Unused Assignment
 
 #define QUIC_API_ENABLE_INSECURE_FEATURES 1
+#define QUIC_PREVIEW_FEATURES 1
```

Comment:
What about matching the format above and do:
```suggestion
#define QUIC_API_ENABLE_PREVIEW_FEATURES 1
```